collections/src/main/scala/strawman/collection/immutable/TreeSet.scala

```diff
@@ -25,11 +25,13 @@ import scala.{Boolean, Int, math, NullPointerException, Option, Ordering, Some,
   *  @define mayNotTerminateInf
   *  @define willNotTerminateInf
   */
+@SerialVersionUID(-5685982407650748405L)
```

Comment:
I started writing a comment here, but actually let's leave it for now and come back to it later :-) https://github.com/scala/collection-strawman/issues/418

Reply:
yeah, when I saw the issue you created it immediately came up to my mind.